xcdat/regridder/accessor.py

```diff
@@ -76,22 +76,15 @@ def grid(self) -> xr.Dataset:
 
         return ds
 
-    def _get_axis_data(
-        self, name: str, standard_name: str
-    ) -> Tuple[xr.DataArray, xr.DataArray]:
-        try:
-            axis = self._ds.cf[name]
-        except KeyError:
-            raise KeyError(
-                f"{standard_name} axis could not be correctly identified in the Dataset"
-            )
+    def _get_axis_data(self, name: CFAxisName) -> Tuple[xr.DataArray, xr.DataArray]:
+        coord_var = get_axis_coord(self._ds, name)
 
         try:
-            axis_bnds = self._ds.bounds.get_bounds(axis.name)
+            bounds_var = self._ds.bounds.get_bounds(name)
```

Comment:
The accepted argument for `bounds.get_bounds()` is now the cf-compliant axis name ("X", "Y", "T", "Z").

You'll notice a bunch of these were updated.